examples/cartpole/README.md

```diff
@@ -3,32 +3,61 @@
+**More CartPole examples can always be added/contributed, using other algorithms/backends.**
 
-## Contents
-- [CartPole discrete](https://github.com/aidudezzz/deepworlds/tree/dev/examples/cartpole/cartpole_discrete)
-- [CartPole continuous](https://github.com/aidudezzz/deepworlds/tree/dev/examples/cartpole/cartpole_continuous)
+----
 
+### Discrete action space
 
-## Solved CartPole - PPO discrete
+The discrete action space case is solved with a custom implementation of the
+[Proximal Policy Optimization Reinforcement (PPO) Learning (RL) algorithm](https://openai.com/blog/openai-baselines-ppo/), 
+both with the [emitter](./cartpole_discrete/controllers/supervisor_manager/supervisor_controller.py)-[receiver](./cartpole_discrete/controllers/robot_controller/robot_controller.py) and 
+[robot supervisor](./cartpole_discrete/controllers/robot_supervisor_manager/robot_supervisor.py) schemes. 
 
-Trained agent showcase:
+Moreover, a solution using 
+[StableBaselines3 is provided](./cartpole_discrete/controllers/robot_supervisor_manager_Stable_Baselines/PPO_runner.py). 
 
-![image](https://github.com/aidudezzz/deepworlds/blob/dev/examples/cartpole/doc/gif/cartpoleSolved.gif)
+You can find the corresponding .wbt world files to open in Webots [here](./cartpole_discrete/worlds/).
+
+----
+
+### Continuous action space
+
+The continuous action space case is solved with a custom implementation of the 
+[Deep Deterministic Policy Gradient RL algorithm](https://arxiv.org/abs/1509.02971), 
+both with the [emitter](./cartpole_continuous/controllers/supervisor_manager/supervisor_controller.py)-[receiver](./cartpole_continuous/controllers/robot_controller/robot_controller.py) and 
+[robot supervisor](./cartpole_continuous/controllers/robot_supervisor_manager/robot_supervisor.py) schemes. 
+
+You can find the corresponding .wbt world files to open in Webots [here](./cartpole_continuous/worlds/).
+
+----
+
+### Contents
+- [CartPole discrete](./cartpole_discrete)
+  - [Emitter](./cartpole_discrete/controllers/supervisor_manager/) / [Receiver](./cartpole_discrete/controllers/robot_controller/) / [Custom PPO](./cartpole_discrete/controllers/supervisor_manager/agent/PPO_agent.py)
+  - [Robot supervisor](./cartpole_discrete/controllers/robot_supervisor_manager) / [Custom PPO](./cartpole_discrete/controllers/robot_supervisor_manager/agent/PPO_agent.py)
+  - [Robot supervisor with StableBaselines3](./cartpole_discrete/controllers/robot_supervisor_manager_Stable_Baselines/)
+- [CartPole continuous](./cartpole_continuous)
+  - [Emitter](./cartpole_continuous/controllers/supervisor_manager/) / [Receiver](./cartpole_continuous/controllers/robot_controller/) / [Custom DDPG](./cartpole_continuous/controllers/supervisor_manager/agent/DDPG_agent.py)
+  - [Robot supervisor](./cartpole_continuous/controllers/robot_supervisor_manager) / [Custom DDPG](./cartpole_continuous/controllers/robot_supervisor_manager/agent/DDPG_agent.py)
```

Comment:
@tsampazk Could you explain why `Receiver` is inside `Emitter`?
Our[ `class SupervisorCSV`](https://github.com/aidudezzz/deepbots/blob/3ae23e6097c2d4ecc4288614981d4114084e6a6e/deepbots/supervisor/controllers/supervisor_emitter_receiver.py#L121) and [`class RobotEmitterReceiverCSV`](https://github.com/aidudezzz/deepbots/blob/3ae23e6097c2d4ecc4288614981d4114084e6a6e/deepbots/robots/controllers/robot_emitter_receiver_csv.py#L7) both have a emitter and a receiver.

Reply:
You're right, i was thinking in terms of the robot being the receiver and the supervisor being the emitter 😆, i'll go ahead and fix this and the next comment appropriately!